### PR TITLE
Vendor keybase/client

### PR DESCRIPTION
**Pull request by akalin-keybase**
This finally removes the last non-test dependency on testing, so
kbfsfuse et al. should now not show testing args in -help!

### CODE REVIEW

File: vendor/vendor.json [hunk 1]
```diff
@@ -155,92 +155,98 @@
 		{
 			"checksumSHA1": "I7uTxTP05oqFbpuwtaPCl0fqElE=",
 			"path": "github.com/keybase/client/go/auth",
-			"revision": "433eb48ff4c25a822386faabcc40fd304269db33",
-			"revisionTime": "2017-10-11T18:56:40Z"
+			"revision": "9fd1a7236d9b727eaa242d8e41496e3b79388785",
```
Comment (strib) [inline vendor/vendor.json]: I don't see this revision in `client/master`.  Can you update it to a real, merged commit?

**Comment by strib** [comment]
giphy-downsized

File: vendor/vendor.json (hunk 1 above)
Comment (akalin-keybase) [reply vendor/vendor.json]: Oops, done! Didn't we have an automated checker for this somewhere?

**Comment by akalin-keybase** [comment]
PTAL, now with less uidmap!

**Pull request merged**